### PR TITLE
Add gatk4 and quanttb combos

### PR DESCRIPTION
**Pull request by abhi18av**

### CODE REVIEW

File: combinations/hash.tsv [hunk 1]
```diff
@@ -172,3 +172,5 @@ gatk=3.5,samtools=1.13
 requests=2.26	quay.io/bioconda/base-glibc-debian-bash:latest	0
 mirtop=0.4.24,samtools=1.13,r-base=4.1.1,r-data.table=1.14.2
 krakenuniq=0.5.8,pigz=2.6
+bioconda::gatk4=4.2.0.0,conda-forge::R=4.1,conda-forge::r-ggplot2=3.3.5,bioconda::datamash=1.1.0
```
Comment (bgruening) [inline combinations/hash.tsv:175]: you don't need to specify the channel

Comment (abhi18av) [reply combinations/hash.tsv:175]: Mmm, I see! 

Thanks for the catch here @bgruening , now I realized that the `abc/xyz` is actually for docker images.

Comment (abhi18av) [reply combinations/hash.tsv:175]:
```suggestion
gatk4=4.2.0.0,R=4.1,r-ggplot2=3.3.5,datamash=1.1.0
```

File: combinations/hash.tsv [hunk 2]
```diff
@@ -172,3 +172,5 @@ gatk=3.5,samtools=1.13
 requests=2.26	quay.io/bioconda/base-glibc-debian-bash:latest	0
 mirtop=0.4.24,samtools=1.13,r-base=4.1.1,r-data.table=1.14.2
 krakenuniq=0.5.8,pigz=2.6
+gatk4=4.2.0.0,R=4.1,r-ggplot2=3.3.5,datamash=1.1.0
+jemunro::quanttb=1.01,bioconda::bwa=0.7.17
```
Comment (abhi18av) [inline combinations/hash.tsv:176]: However, for the `quanttb` package - would the build system pick it up without the `jemunro` ?

Comment (bgruening) [reply combinations/hash.tsv:176]: Oh I see, no. But we also do not support that. Please move the package to bioconda or conda-forge.

Comment (abhi18av) [reply combinations/hash.tsv:176]: Got it, so for the time being, I'll remove that line from the PR.

**Comment by abhi18av** [comment]
@bgruening , I've resolved the merge conflict - perhaps we can continue with the merge?

**Comment by bgruening** [comment]
Will this also work?

**Comment by abhi18av** [comment]
I think so - basically it's the equivalent of `conda-forge::R=4.1` right?

**Pull request merged**